otelcol.processor.k8sattributes: Add example with labels of Prometheus metrics (#5632)
grafana/agent: docs/sources/flow/reference/components/otelcol.processor.k8sattributes.md

  }
}
```

### Adding Kubernetes metadata to Prometheus metrics

`otelcol.processor.k8sattributes` adds metadata to metrics signals in the form of resource attributes.

> In order to display the metadata as labels of Prometheus metrics, the OTLP attributes need to be converted from

Reviewer: ```suggestion
To display the metadata as labels of Prometheus metrics, the OTLP attributes must be converted from
```
